### PR TITLE
Feat/add timechecker for moment comparison

### PR DESCRIPTION
**Pull request by AlecDWalker**
Refactor to remove an unneeded import. Think I forgot to add this commit last time?

Added a bool to the timeScreenVM class to track alarm status.

Added a test to make sure that a new screen begins with the alarm set to false.

Might refactor into a helper file later (e.g. 'alarmChecker'?)

### CODE REVIEW

File: components/__tests__/alarmBoolean-test.js
Comment (Edward-Phillips) [inline components/__tests__/alarmBoolean-test.js]: good to see you're disposing! As it's being used in two tests I would probably instantiate the timescreen a new time for each test, so move line 4 to lines 7 and 15, and duplicate line 23 on line 11

File: viewModels/timeScreenVM/timeScreenVM.tsx
Comment (Edward-Phillips) [inline viewModels/timeScreenVM/timeScreenVM.tsx:51]: nice and simple!

**Review by Edward-Phillips — COMMENTED** [review]
looks good! It looks like there are some conflicts to resolve between this branch and master which you'll need to look at before this can be merged though.

 Are the tests passing? We can add a ticket later to run the tests automatically when a PR is made but no worries for the time being.

**Comment by AlecDWalker** [comment]
> looks good! It looks like there are some conflicts to resolve between this branch and master which you'll need to look at before this can be merged though.
> 
> Are the tests passing? We can add a ticket later to run the tests automatically when a PR is made but no worries for the time being.

Yep, tests are all passing so far. And from what I can see the conflict is just an extra curly bracket that needs removing

**Comment by AlecDWalker** [comment]
I think I'm good to merge if you're giving the all clear?

**Review by Edward-Phillips — APPROVED** [review]
LGTM

**Pull request merged**
